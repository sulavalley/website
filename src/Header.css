.header-size{
    height: 85px;
}

img{
    width: 195px;
    height: auto;
    margin-left: 2%;
}