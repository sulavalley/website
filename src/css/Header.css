img {
  width: 10%;
  margin-left: 2%;
}

@media only screen and (min-device-width: 422px) and (max-device-width: 976px) {
  img {
    width: 32%;
  }
}

@media only screen and (max-width: 421px) {
  img {
    width: 40%;
  }
}
