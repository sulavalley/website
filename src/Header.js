import React from 'react'
import './Header.css'

const Header = () => {
    return (
        <div className="ui header-size menu">
            <img src="https://trello-attachments.s3.amazonaws.com/5aff552929693747ec0b088d/5aff5c23f3cddefc5870f975/9641415fc09e4d3aa2b1d3fd734b9541/logo.png" />
            <div className="right menu">
                <a className="item" href="/institucionalidad">Institucionalidad</a>
                <a className="item" href="/municipios">Municipios</a>
                <a className="item" href="/multimedia">Multimedia</a>
                <a className="item" href="/sostenibilidad">Sostenibilidad</a>
                <a className="item" href="/festividades">Festividades</a>
                <a className="item" href="/actividades">Actividades</a>
                <a className="item" href="/blog">Blog</a>
            </div>
        </div>
    );
};

export default Header;