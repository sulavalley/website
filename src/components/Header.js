import React, { Component } from 'react';
import './../css/Header.css';
import logo from './../assets/images/logo.png';
import SignIn from './SignIn';
import { auth } from '../base';
import CurrentUser from './CurrentUser';

class Header extends Component {
  state = {
    currentUser: null
  };
  componentDidMount() {
    auth.onAuthStateChanged(currentUser => {
      this.setState({ currentUser });
    });
  }
  render() {
    const { currentUser } = this.state;
    return (
      <div className="ui header-size menu">
        <img src={logo} alt="logo" />
        <div className="right menu">
          <a className="item" href="/institucionalidad">
            Institucionalidad
          </a>
          <a className="item" href="/municipios">
            Municipios
          </a>
          <a className="item" href="/multimedia">
            Multimedia
          </a>
          <a className="item" href="/sostenibilidad">
            Sostenibilidad
          </a>
          <a className="item" href="/festividades">
            Festividades
          </a>
          <a className="item" href="/actividades">
            Actividades
          </a>
          <a className="item" href="/blog">
            Blog
          </a>
          {!currentUser && <SignIn />}
          {currentUser && <CurrentUser user={currentUser} />}
        </div>
      </div>
    );
  }
}

export default Header;
