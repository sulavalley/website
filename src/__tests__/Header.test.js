import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './../Header';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Header />);
});

test('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
