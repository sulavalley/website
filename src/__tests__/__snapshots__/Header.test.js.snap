// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui header-size menu"
>
  <img
    alt="logo"
    src="logo.png"
  />
  <div
    className="right menu"
  >
    <a
      className="item"
      href="/institucionalidad"
    >
      Institucionalidad
    </a>
    <a
      className="item"
      href="/municipios"
    >
      Municipios
    </a>
    <a
      className="item"
      href="/multimedia"
    >
      Multimedia
    </a>
    <a
      className="item"
      href="/sostenibilidad"
    >
      Sostenibilidad
    </a>
    <a
      className="item"
      href="/festividades"
    >
      Festividades
    </a>
    <a
      className="item"
      href="/actividades"
    >
      Actividades
    </a>
    <a
      className="item"
      href="/blog"
    >
      Blog
    </a>
    <a
      className="item"
      onClick={[Function]}
    >
      Iniciar Sesi√≥n
    </a>
  </div>
</div>
`;
